src/formatters/deployResultFormatter.ts

```diff
@@ -7,39 +7,42 @@
 
 import * as path from 'path';
 import * as chalk from 'chalk';
-import { UX } from '@salesforce/command';
-import { Logger, Messages, SfError } from '@salesforce/core';
+
+import { Messages, SfError } from '@salesforce/core';
 import { ensureArray } from '@salesforce/kit';
 import { asString, get, getBoolean, getNumber, getString } from '@salesforce/ts-types';
```

Comment:
I really don't like the TS-abuse happening with getNumber/getString below.  I can't comment on the appropriate lines because they're not part of this PR, but a large-scale testing blitz would be a good time to fix the types.

getResponse() will return a {} but say that it matches the return type.